### PR TITLE
Initial implementation of subprojects

### PR DESCRIPTION
**Pull request by spiffxp**
Initial implementation of #1673

A subproject is a one-to-many mapping of a name to a list of
OWNERS urls.  This allows for a subproject to be a single repo (eg:
helm), or a subproject to be a list of multiple subdirectories across
one or more repos (eg: workloads api)

Strawman assignment of single repo subprojects per an initial mapping
from @bgrant0607

### CODE REVIEW

**Comment by jdumars** [comment]
Initial review looks very good. I didn't see any glaring misalignments.

**Comment by cblecker** [comment]
My main concern with how this is implemented/displayed is that is balloons the size/length of the document, pushing the custom content and github teams sections really far down the page.

Perhaps if we have defined fields, this could be better displayed as a table?

**Comment by spiffxp** [comment]
Yeah I'm not happy with the formatting atm, table sounds like a good idea

File: sigs.yaml [hunk 1]
```diff
@@ -90,6 +166,53 @@ sigs:
         description: Design Proposals
       - name: sig-apps-test-failures
         description: Test Failures and Triage
+    subprojects:
```
Comment (mattfarina) [inline sigs.yaml]: I would suggest adding the Workloads API and pointing to the OWNERS files for the controllers. It's an opportunity to build for a sub-project with multiple OWNERS files.

File: sig-testing/README.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Interested in how we can most effectively test Kubernetes. We're interested spec
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-testing)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Ftesting)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **test-infra**
```
Comment (fejta) [inline sig-testing/README.md]: LGTM

File: generator/sig_readme.tmpl [hunk 3]
```diff
@@ -25,6 +25,35 @@
 {{- if .Label }}
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2F{{.Label}})
 {{- end }}
+{{- if .Subprojects }}
+
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (fejta) [inline generator/sig_readme.tmpl]: `sig%2F{{.Label}}`?

**Comment by bgrant0607** [comment]
I think this is a great start. Formatting could be changed/improved on a separate pass. 

LGTM

**Comment by thockin** [comment]
My concern is that this will become stale.  It's so far removed from the repos that it will be easy to forget - what forces me to update this?  It sort of goes the opposite direction of Go's "convention over configuration" ethos.

Don't get me wrong - I think we need something like this, but what stops it from being wrong?

**Comment by bgrant0607** [comment]
@thockin This is a first step. Automation will be built to keep this up to date.

**Comment by bgrant0607** [comment]
@spiffxp also has/had work in flight to add subprojects to the appropriate OWNERS files

**Comment by spiffxp** [comment]
/hold
holding for further comment

File: subprojects.json [hunk 4]
```diff
@@ -0,0 +1,548 @@
+[
```
Comment (cblecker) [inline subprojects.json]: Is this artifact used?

File: subprojects.csv [hunk 5]
```diff
@@ -0,0 +1,91 @@
+kubernetes,api,api-machinery
```
Comment (cblecker) [inline subprojects.csv]: same with this

File: sigs.yaml [hunk 6]
```diff
@@ -90,6 +166,63 @@ sigs:
...
+    - name: chart-testing
+      owners:
+        - https://github.com/kubernetes-helm/chart-testing/blob/master/OWNERS
+    - name: chartmuseum
+      owners:
+        - https://github.com/kubernetes-helm/chartmuseum/blob/master/OWNERS
+    - name: charts
+      meetings:
```
Comment (cblecker) [inline sigs.yaml]: Does this mean meetings are in two different places?

File: subprojects.csv (hunk 5 above)
Comment (spiffxp) [reply subprojects.csv]: derp don't know how this got checked in, thanks

File: subprojects.json (hunk 4 above)
Comment (spiffxp) [reply subprojects.json]: derp don't know how this got checked in, thanks

File: sigs.yaml (hunk 1 above)
Comment (spiffxp) [reply sigs.yaml]: done

File: sigs.yaml (hunk 6 above)
Comment (spiffxp) [reply sigs.yaml]: the idea is that subprojects may have their own meetings independent of the entire sig

Comment (cblecker) [reply sigs.yaml]: Which makes sense, but for example, the helm/charts calls are already defined elsewhere in sigs.yaml (https://github.com/kubernetes/community/blob/02d7d8e61aaf888a583b813054681f61424f8b06/sigs.yaml#L63-L85), and the templates know how to look for those. Perhaps there just needs to be a way to associate subprojects with the meetings?

Biggest objection is to defining meeting information in two places

Comment (spiffxp) [reply sigs.yaml]: Oops, I meant to move those meetings down into the subprojects, and forgot (I assumed there'd be a merge conflict on rebase)... but it does mean you don't see all meetings for a SIG and all of its subprojects at a glance in sigs-list.md

I'm open to changing one way or another but I'm going to default to DRY meetings defined only for the relevant subproject or SIG and await further feedback

**Comment by spiffxp** [comment]
[kubernetes-dev thread](https://groups.google.com/forum/#!topic/kubernetes-dev/1aBR4knQ8rQ)

I will merge Tuesday Feb 6th if there are no objections or comments that can't be properly addressed

File: sig-list.md [hunk 7]
```diff
@@ -48,7 +48,7 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
...
 |[Storage](sig-storage/README.md)|storage|* [Saad Ali](https://github.com/saad-ali), Google<br>* [Bradley Childs](https://github.com/childsb), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-storage)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-storage)|* Regular SIG Meeting: [Thursdays at 9:00 PT (Pacific Time) (biweekly)](https://zoom.us/j/614261834)<br>
-|[Testing](sig-testing/README.md)|testing|* [Aaron Crickenberger](https://github.com/spiffxp), Samsung SDS<br>* [Erick Feja](https://github.com/fejta), Google<br>* [Steve Kuznetsov](https://github.com/stevekuznetsov), Red Hat<br>* [Timothy St. Clair](https://github.com/timothysc), Heptio<br>|* [Slack](https://kubernetes.slack.com/messages/sig-testing)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-testing)|* Regular SIG Meeting: [Tuesdays at 13:00 PT (Pacific Time) (weekly)](https://zoom.us/my/k8s.sig.testing)<br>* Testing Commons: [Wednesdays at 07:30 PT (Pacific Time) (bi-weekly)](https://zoom.us/my/k8s.sig.testing)<br>
```
Comment (cblecker) [inline sig-list.md]: Should subproject meetings be listed in sig-list.md?

File: sig-api-machinery/README.md [hunk 8]
```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+- **community**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/community/master/OWNERS
+- **csharp**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/csharp/master/OWNERS
+- **gen**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/gen/master/OWNERS
+- **gengo**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: I'd include gengo in an "IDL" subproject.

File: sig-api-machinery/README.md [hunk 9]
```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+- **haskell**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/haskell/master/OWNERS
+- **java**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/java/master/OWNERS
+- **javascript**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/javascript/master/OWNERS
+- **kube-aggregator**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: API Aggregation subproject

File: sig-api-machinery/README.md [hunk 10]
```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kube-aggregator/master/OWNERS
+- **kube-openapi**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: I'd include openapi in the client lib subproject

File: sig-api-machinery/README.md [hunk 11]
```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kube-openapi/master/OWNERS
+- **python-base**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/python-base/master/OWNERS
+- **ruby**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/ruby/master/OWNERS
+- **sample-apiserver**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: sample-apiserver and sample-controller would make sense in the apiserver subproject.

File: sig-api-machinery/README.md [hunk 12]
```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/sample-apiserver/master/OWNERS
+- **sample-controller**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/sample-controller/master/OWNERS
+- **typescript**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/typescript/master/OWNERS
+- **utils**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: utils is its own subproject

File: sig-apps/README.md [hunk 13]
```diff
@@ -29,6 +25,56 @@ Covers deploying and operating applications in Kubernetes. We focus on the devel
...
+  - Meetings:
+    - Charts Chat: [Tuesdays at 9:00 PT (Pacific Time)](https://zoom.us/j/166909412) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+      - [Meeting notes and Agenda](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
+- **charts-tooling**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/charts-tooling/master/OWNERS
+- **community**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/community/master/OWNERS
+- **examples**
```
Comment (bgrant0607) [inline sig-apps/README.md]: We should ask the Helm community whether 2 subprojects are enough, or whether approx. 1 subproject per repo is needed, but could we at least group the chart-related and helm-related repos together in this list?

File: sig-architecture/README.md [hunk 14]
```diff
@@ -24,6 +24,22 @@ The Architecture SIG maintains and evolves the design principles of Kubernetes,
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-architecture)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Farchitecture)
 
+## Subprojects
```
Comment (bgrant0607) [inline sig-architecture/README.md]: Actually, I think k8s.io maybe should be contribX and the rest should be owned by the steering committee.

SIG Arch efforts are listed here:
https://github.com/kubernetes/community/blob/master/sig-architecture/charter.md

Many of the items clearly need to be moved so that they can have dedicated owners files.

File: sig-big-data/README.md [hunk 15]
```diff
@@ -24,6 +24,13 @@ Covers deploying and operating big data applications (Spark, Kafka, Hadoop, Flin
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-big-data)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fbig-data)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **application-images**
```
Comment (bgrant0607) [inline sig-big-data/README.md]: This was retired, no?

File: sig-cli/README.md [hunk 16]
```diff
@@ -25,6 +25,16 @@ Covers kubectl and related tools. We focus on the development and standardizatio
...
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **common**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/common/master/OWNERS
+- **kubectl**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kubectl/master/OWNERS
+
```
Comment (bgrant0607) [inline sig-cli/README.md]: Kubectl's main location is in the kubernetes repo:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/OWNERS

File: sig-cluster-lifecycle/README.md [hunk 17]
```diff
@@ -26,6 +26,40 @@ The Cluster Lifecycle SIG examines how we should change Kubernetes to make it ea
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fcluster-lifecycle)
 
+## Subprojects
```
Comment (bgrant0607) [inline sig-cluster-lifecycle/README.md]: Also:

kubeadm:
https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm

kube-up:
https://github.com/kubernetes/kubernetes/tree/master/cluster

File: sig-contributor-experience/README.md [hunk 18]
```diff
@@ -28,6 +28,16 @@ Developing and sustaining a healthy community of contributors is critical to sca
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-contribex)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fcontributor-experience)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **community**
```
Comment (bgrant0607) [inline sig-contributor-experience/README.md]: LG

File: sig-docs/README.md [hunk 19]
```diff
@@ -24,6 +24,22 @@ Covers documentation, doc processes, and doc publishing for Kubernetes.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fdocs)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-docs/README.md]: LG

File: sig-instrumentation/README.md [hunk 20]
```diff
@@ -23,6 +23,22 @@ Covers best practices for cluster observability through metrics, logging, and ev
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-instrumentation)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Finstrumentation)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-instrumentation/README.md]: LG

File: sig-list.md [hunk 21]
```diff
@@ -23,7 +23,7 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
...
-|[Apps](sig-apps/README.md)|apps|* [Matt Farina](https://github.com/mattfarina), Samsung SDS<br>* [Adnan Abdulhussein](https://github.com/prydonius), Bitnami<br>* [Kenneth Owens](https://github.com/kow3ns), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-apps)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-apps)|* Regular SIG Meeting: [Mondays at 9:00 PT (Pacific Time) (weekly)](https://zoom.us/my/sig.apps)<br>* Helm Developer call: [Thursdays at 9:30 PT (Pacific Time) (weekly)](https://zoom.us/j/4526666954)<br>* Charts Chat: [Tuesdays at 9:00 PT (Pacific Time) (biweekly)](https://zoom.us/j/166909412)<br>
+|[Apps](sig-apps/README.md)|apps|* [Matt Farina](https://github.com/mattfarina), Samsung SDS<br>* [Adnan Abdulhussein](https://github.com/prydonius), Bitnami<br>* [Kenneth Owens](https://github.com/kow3ns), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-apps)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-apps)|* Regular SIG Meeting: [Mondays at 9:00 PT (Pacific Time) (weekly)](https://zoom.us/my/sig.apps)<br>
```
Comment (bgrant0607) [inline sig-list.md]: Was this change deliberate?

File: sig-multicluster/README.md [hunk 22]
```diff
@@ -24,6 +24,16 @@ A Special Interest Group focussed on solving common challenges related to the  m
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-multicluster)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fmulticluster)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **cluster-registry**
```
Comment (bgrant0607) [inline sig-multicluster/README.md]: LG

File: sig-network/README.md [hunk 23]
```diff
@@ -25,6 +25,25 @@ Covers networking in Kubernetes.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-network)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fnetwork)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-network/README.md]: The subprojects I suggest are:
services, including kube-proxy
ingress
kube-dns
external-dns
pod networking: CNI, ip-masq-agent, etc.
network policy

File: sig-node/README.md [hunk 24]
```diff
@@ -23,6 +23,31 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fnode)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-node/README.md]: kubelet, obviously
https://github.com/kubernetes/kubernetes/tree/master/cmd/kubelet
https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet

File: sig-release/README.md [hunk 25]
```diff
@@ -23,6 +23,16 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Frelease)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **release**
```
Comment (bgrant0607) [inline sig-release/README.md]: I'd add hyperkube as a subproject:
https://github.com/kubernetes/kubernetes/tree/master/cmd/hyperkube

File: sig-scalability/README.md [hunk 26]
```diff
@@ -26,6 +26,13 @@ For more details about our objectives please review our [Scaling And Performance
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-scale)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fscalability)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **perf-tests**
```
Comment (bgrant0607) [inline sig-scalability/README.md]: kubemark:
https://github.com/kubernetes/kubernetes/tree/master/cmd/kubemark
https://github.com/kubernetes/kubernetes/tree/master/pkg/kubemark

File: sig-scheduling/README.md [hunk 27]
```diff
@@ -23,6 +23,19 @@ To understand how this file is generated, see https://git.k8s.io/community/gener
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-scheduling)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fscheduling)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-scheduling/README.md]: scheduler:
https://github.com/kubernetes/kubernetes/tree/master/cmd/kube-scheduler
https://github.com/kubernetes/kubernetes/tree/master/pkg/scheduler

File: sig-storage/README.md [hunk 28]
```diff
@@ -24,6 +24,19 @@ Covers storage and volume plugins.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-storage)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fstorage)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
```
Comment (bgrant0607) [inline sig-storage/README.md]: volumes:
https://github.com/kubernetes/kubernetes/tree/master/pkg/volume

File: sig-testing/README.md (hunk 2 above)
Comment (bgrant0607) [reply sig-testing/README.md]: What about test infra in the kubernetes repo, such as parts of https://github.com/kubernetes/kubernetes/tree/master/test/e2e?

File: sig-windows/README.md [hunk 29]
```diff
@@ -23,6 +23,13 @@ Focuses on supporting Windows Server Containers for Kubernetes.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-windows)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fwindows)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **dashboard**
```
Comment (bgrant0607) [inline sig-windows/README.md]: This should be sig-ui rather than sig-windows

File: sig-api-machinery/README.md (hunk 12 above)
Comment (spiffxp) [reply sig-api-machinery/README.md]: under which sig?

File: sig-architecture/README.md (hunk 14 above)
Comment (spiffxp) [reply sig-architecture/README.md]: steering committee doesn't yet exist as a formalized concept of a group as this file / generator are concerned, so I'd like to defer moving to it until then

File: sig-list.md (hunk 21 above)
Comment (spiffxp) [reply sig-list.md]: yes, the idea is that sig-level meetings are punted down into subproject-level meetings, which aren't listed in this table right now, I'll look into redoing the formatting to add them back in if it's not too cramped

File: sig-network/README.md (hunk 23 above)
Comment (spiffxp) [reply sig-network/README.md]: I need OWNERS files to accompany these, I'll take a best guess for now

File: sig-apps/README.md (hunk 13 above)
Comment (mattfarina) [reply sig-apps/README.md]: To address Brians feedback here's my 2 cents as the groupings can be quite subjective.

* _chart-tooling_ can be grouped with _charts_. That is supporting one off tooling experiments.
* _community_ can be grouped under _helm_ as that is supporting docs.

After that things get a little less clear:

* _monocular_ is worked on by _helm_ and _charts_ folks and leverages code and work from both of those. But, it is it's own application. The same can be said of _chartmuseum_. They could be on their own or they could be lumped under _helm_.
* Depending on how you answer the previous one, _rudder-federation_ and _chart-testing_ could be grouped as individual efforts or separate sub-projects.

A lot of this comes down to what is supporting what and where to break things apart. Below is a tree of how, I think, things relate to each other in the Helm space.

```
                                  +---------+
                                  |  Helm   |
     +----------------------+-----+--------++-------------------+--------------------------+---------------------+
     |                      |              |                    |                          |                     |
     |                      |              |                    |                          |                     |
+----v------+           +---v----+---------+          +---------v----------+        +------v-------+     +-------v-------+
| Community |           | Charts |         |          |  Rudder Federation |        | Chart Museum |     | Chart Testing |
+-----------+    +------+--------+         |          +--------------------+        +--------------+     +---------------+
                 |                   +-----v-----+
                 |                   | Monocular |
                 |                   +-----------+
        +--------v-------+
        | Charts Tooling |
        +----------------+
```

At what level are sub-projects broken apart into smaller bits?

File: sigs.yaml [hunk 30]
```diff
@@ -101,6 +163,63 @@ sigs:
...
+    - name: kompose
+      owners:
+      - https://raw.githubusercontent.com/kubernetes/kompose/master/OWNERS
+    - name: monocular
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/monocular/master/OWNERS
+    - name: rudder-federation
+      owners:
+      - https://raw.githubusercontent.com/kubernetes-helm/rudder-federation/master/OWNERS
+    - name: workloads-api
```
Comment (mattfarina) [inline sigs.yaml]: jobs and cronjobs should also be here.

File: sig-apps/README.md (hunk 13 above)
Comment (spiffxp) [reply sig-apps/README.md]: I've done the first two groupings.  We can iterate on the rest as needed.  My guess was going to be "everything in kuberentes-helm is part of helm."  Personally I don't see a ton of value in defining turtles all the way down unless there are disjoint sets of maintainers and consumers.

Comment (technosophos) [reply sig-apps/README.md]: I am not opposed to grouping Helm into one project (all the things!) or two projects (Helm code, charts).

In the past, we've tried to make a governance distinction between charts and Helm in order to prevent developers from making ops decisions. So we've talked about them as truly distinct projects. But org-wise, maybe it would actually make sense to group them.

File: sig-testing/README.md (hunk 2 above)
Comment (spiffxp) [reply sig-testing/README.md]: For now I have put all of k/k/test under the "testing-commons" subproject, rather than sharding by subfolder

File: sig-apps/README.md (hunk 13 above)
Comment (mattfarina) [reply sig-apps/README.md]: @spiffxp If we group things by maintainers you have:

* Helm
* Charts
* Chart Museum

While there are some overlaps those are the most distinct groups of maintainers.

The reason I described turtles all the way down was for clarity. We don't really need to define that for this doc.

**Comment by spiffxp** [comment]
I will take a crack at adding the subproject meetings to sig-list.md later today but honestly I think the formatting on that page needs to be re-thought and I don't want to play with it too much in this PR.

File: sigs.yaml (hunk 30 above)
Comment (spiffxp) [reply sigs.yaml]: Done, I updated the description and added the OWNERS files for the controllers and the batch api

**Comment by spiffxp** [comment]
/hold cancel
ready for /lgtm

**Comment by bgrant0607** [comment]
Awesome. Merge and iterate. In the next attempt, I can try to enumerate more of the subprojects in the kubernetes repo.

/lgtm

File: sig-api-machinery/README.md [hunk 31]
```diff
@@ -24,6 +24,53 @@ Covers all aspects of API server, API registration and discovery, generic API CR
...
+    - https://raw.githubusercontent.com/kubernetes-incubator/apiserver-builder/master/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/sample-apiserver/master/OWNERS
+    - https://raw.githubusercontent.com/kubernetes/sample-controller/master/OWNERS
+- **code-generator**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/code-generator/master/OWNERS
+- **crd**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/apiextensions-apiserver/master/OWNERS
+- **gen**
```
Comment (bgrant0607) [inline sig-api-machinery/README.md]: Is this gengo?

Comment (bgrant0607) [reply sig-api-machinery/README.md]: No, that's IDL, below
